container.go

```diff
@@ -66,77 +66,117 @@ func getCRIOInfo(_ *metadata.ContainerConfig, specDump *spec.Spec) (*containerIn
 	}, nil
 }
 
-func showContainerCheckpoint(checkpointDirectory, input string) error {
-	containerConfig, _, err := metadata.ReadContainerCheckpointConfigDump(checkpointDirectory)
-	if err != nil {
-		return err
+func showContainerCheckpoints(tasks []task) error {
+	// Return an error early when attempting to display multiple checkpoints with mount or stats flags
+	if (len(tasks) > 1) && (mounts || stats) {
+		return fmt.Errorf("displaying multiple checkpoints with mount or stats flags is not supported")
```

Comment:
There might be a few more flags added to checkpointctl, and it might be better to use a more generic error message instead of listing all flags (e.g., "displaying multiple checkpoints with _additional_ flags is not supported")